packages/frontend/src/components/campaigns/rewards/index.tsx

```diff
@@ -21,7 +21,9 @@ export function Rewards({ from, to, rewards }: RewardsProps) {
           dayjs.unix(to).diff(dayjs.unix(from), "days", false)
         : 0;
 
-    return (
+    return perDayUsdValue === 0 ? (
+        <Typography>-</Typography>
```

Comment:
We should add `weight="medium"` just like the apr.

Reply:
Addressed in 16addf02c5a29363d358321ab31bbd09d3976d49